tests/system/Storage/StreamWrapper/UrlStatTest.php

```diff
@@ -74,8 +74,8 @@ public function testIsReadable()
 
     public function testFileExists()
     {
-        $this->assertTrue(file_exists(self::$dirUrl));
-        $this->assertTrue(file_exists(self::$fileUrl));
+        $this->assertFilExists(self::$dirUrl);
```

Comment:
mis-typed.

Reply:
Oh, sorry about that. Gonna fix it :flushed: